### PR TITLE
modify getMetadata to have same results as others services

### PR DESCRIPTION
**Pull request by JacquesBorsenberger**
Signed-off-by: jacques borsenberger <jacques.borsenberger@rte-france.com>

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/repository/StudyRepository.java [hunk 1]
```diff
@@ -21,8 +21,8 @@
 @Repository
 public interface StudyRepository extends JpaRepository<StudyEntity, UUID> {
 
-    @Query(value = "SELECT * FROM study u WHERE id in ?1 and (is_private='false' or user_id=?2)", nativeQuery = true)
-    List<StudyEntity> findAllByUuids(List<UUID> uuids, String userId);
+    @Query(value = "SELECT * FROM study u WHERE id in ?1", nativeQuery = true)
+    List<StudyEntity> findAllByUuids(List<UUID> uuids);
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/study/server/repository/StudyRepository.java]: Native request not necessary ?

File: src/test/java/org/gridsuite/study/server/StudyTest.java [hunk 2]
```diff
@@ -827,6 +805,34 @@ public void test() {
...
+                .header("userId", "userId")
+                .exchange()
+                .expectStatus().isOk()
+                .expectHeader().contentType(MediaType.APPLICATION_JSON)
+                .expectBodyList(CreatedStudyBasicInfos.class)
+            .returnResult().getResponseBody();
+
+        assertNotNull(res);
+        assertEquals(2, res.size());
+        if (!res.get(0).getId().equals(oldStudyUuid)) {
```
Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/study/server/StudyTest.java:829]: Sort the list ?

**Review by SlimaneAmar — APPROVED** [review]

File: src/main/java/org/gridsuite/study/server/repository/StudyRepository.java (hunk 1 above)
Comment (JacquesBorsenberger) [reply src/main/java/org/gridsuite/study/server/repository/StudyRepository.java]: removed

File: src/test/java/org/gridsuite/study/server/StudyTest.java (hunk 2 above)
Comment (JacquesBorsenberger) [reply src/test/java/org/gridsuite/study/server/StudyTest.java:829]: on what ?

**Pull request merged**